dvc/repo/experiments/executor/base.py

```diff
@@ -248,6 +250,27 @@ def on_diverged_ref(orig_ref: str, new_rev: str):
             )
         return refs
 
+    @classmethod
+    def _validate_remotes(cls, dvc: "Repo"):
+        git_remote = os.getenv(DVC_EXP_GIT_REMOTE, None)
+        if git_remote == dvc.root_dir:
+            logger.warning(
+                f"'{git_remote}' points to the current Git repo, experiment "
+                "Git refs will not be pushed. But DVC cache and run cache "
+                "will automatically be pushed to the default DVC remote "
+                "(if any) on each experiment commit."
+            )
+        try:
+            for ref in dvc.scm.iter_remote_refs(
+                git_remote, base=EXPS_NAMESPACE
+            ):
+                if ref:
+                    break
+        except BaseException as e:
+            raise e
```

Comment:
This try/except is unnecessary (nothing is actually done in the except block other than re-raising the exception)